config/event_handling.tmpl.yml

```diff
@@ -532,14 +534,6 @@ event_groups:
     - NODE_REBOOT
     - NODE_NODESCHEDULABLE
     - NODE_NODENOTSCHEDULABLE
-    - POD_SCHEDULED
-    - POD_FAILEDSCHEDULING
-    - REPLICATOR_SUCCESSFULCREATE
-    - REPLICATOR_FAILEDCREATE
-    - CONTAINER_CREATED
-    - CONTAINER_KILLING
-    - CONTAINER_STARTED
-    - CONTAINER_STOPPED
```

Comment:
I think ```CONTAINER_STARTED``` and ```CONTAINER_STOPPED``` should stay under ```power```, in similar fashion as for vms (start, stop and pause)